### PR TITLE
#167729781 Setup API documentation using swagger

### PR DESCRIPTION
**Pull request by okafor-chidimma**
#### What does this PR do?

Creates a swagger.json file and other packages for api documentation

#### Description of Task to be completed?

Create api documentation page using Swagger that is easily accessible to a User

#### How should this be manually tested?

Navigate to the url hosting the documentation to view it

#### Any background context you want to provide?

Ensure that you are using the accurate path

#### What are the relevant pivotal tracker stories?

#167729781

#### Screenshots (if appropriate)
None
#### Questions:

No

### CODE REVIEW

**Comment by okafor-chidimma** [comment]
@okabamac @sebaztyn @mthamayor Please review and approve for merge or feedback. Thank you

**Review by mthamayor — APPROVED** [review]

**Review by mthamayor — APPROVED** [review]
Nice work with this chore

**Review by okabamac — APPROVED** [review]
Looks great 👍

**Comment by chibytez** [comment]
nice one

**Review by chibytez — APPROVED** [review]

File: swagger.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0",
+    "title": "BAREFOOT NOMAD",
+    "description": "Making company tavel and accomodation easy and convinient"
```
Comment (Stylll) [inline swagger.json]: Hi @okafor-chidimma , please update the spellings in the description text. Some words are not spelled properly.

**Review by Stylll — CHANGES_REQUESTED** [review]

File: swagger.json (hunk 1 above)
Comment (okafor-chidimma) [reply swagger.json]: @Stylll Please review, I have implemented the changes

Comment (Stylll) [reply swagger.json]: @okafor-chidimma I can't see the changes implemented.

**Review by Stylll — CHANGES_REQUESTED** [review]

**Review by mthamayor — CHANGES_REQUESTED** [review]
https://github.com/andela/shadowcat-backend/blob/669ad8c0ff4869521a2632c5b343f8c72c6936d8/swagger.json#L6

Please check the spelling on this line

**Review by okabamac — APPROVED** [review]
Looks great now

File: swagger.json (hunk 1 above)
Comment (okafor-chidimma) [reply swagger.json]: @Stylll Please check the last commit. I had to push again

**Review by mthamayor — APPROVED** [review]

**Review by Stylll — APPROVED** [review]

**Pull request merged**